rlp/src/stream.rs

```diff
@@ -344,6 +344,12 @@ impl RlpStream {
 		self.note_appended(1);
 		self.finished_list = true;
 	}
+
+	/// Deprecated, use finalize_unbounded_list instead.
```

Comment:
Let's keep the documentation, but [use the deprecation attribute](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute) instead
```suggestion
	/// Finalize current unbounded list. Panics if no unbounded list has been opened.
	#[deprecated(since = "0.4.3", reason = "use finalize_unbounded_list instead")]
```

Reply:
Thanks, updated!